targettests/Remote-Sim/observe.cpp

```diff
@@ -26,7 +24,11 @@ struct ansatz {
     cudaq::qvector q(2);
     x(q[0]);
     ry(theta, q[1]);
+#if CUDAQ_USE_STD20
```

Comment:
Same

Reply:
Fixed in https://github.com/NVIDIA/cuda-quantum/pull/1435/commits/4ad7ecd84a4f0d7c8dac820095463a5234365331